lld/test/COFF/wrap-inherit-weakness.ll

```diff
@@ -0,0 +1,16 @@
+; REQUIRES: x86
+
+; Check that 'weak' attribute will be inherited to the wrapped symbols.
```

Comment:
Can you elaborate a bit more about what happens here? I don't quite understand the involved mechanics here.